api/src/main/java/org/pmiops/workbench/api/UserAdminController.java

```diff
@@ -98,8 +95,9 @@ public ResponseEntity<EmptyResponse> unsafeSelfBypassAccessRequirement(
     if (!workbenchConfigProvider.get().access.unsafeAllowSelfBypass) {
       throw new ForbiddenException("Self bypass is disallowed in this environment.");
     }
-    long userId = userProvider.get().getUserId();
-    userService.updateBypassTime(userId, request);
+    final DbUser user = userProvider.get();
+    accessModuleService.updateBypassTime(user.getUserId(), request);
```

Comment:
TODO soon: call `updateUserAccessTiers()` from `updateBypassTime()` and `updateCompletionTime()`